linera-core/src/worker.rs

```diff
@@ -302,6 +315,17 @@ where
             .await
     }
 
+    #[inline]
+    pub(crate) async fn fully_handle_lite_certificate_with_notifications(
```

Comment:
Do we need this function if it is used only once and only calls two other functions?